src/services/school/model.js

```diff
@@ -47,7 +54,11 @@ const schoolSchema = new Schema({
 });
 
 const yearSchema = new Schema({
-	name: { type: String, required: true },
+	name: {
+		type: String, required: true, match: /^[0-9]{4}\/[0-9]{2}$/, unique: true,
+	},
+	startDate: { type: Date, required: true },
```

Comment:
should we index these? The workload will be 99.9% reads

Reply:
normally all years will be loaded, there is no search on names, and _id should have an index.